rainier-core/src/main/scala/com/stripe/rainier/compute/RealOps.scala

```diff
@@ -90,6 +90,12 @@ private[compute] object RealOps {
         LogLineOps.multiply(LogLine(nc1), LogLine(nc2))
     }
 
+  def divide(left: Real, right: Real): Real =
+    (left, right) match {
+      case (Constant(x), Constant(y)) => Real(x / y)
```

Comment:
Actually, in the repl, `1.0 / 0.0` gives `Infinity` whereas `BigDecimal(1.0) / BigDecimal(0.0)` throws an exception. We should guard against this.

Reply:
Yep, `Real(1.0) / Real(0.0)` throws an exception.

Reply:
Addressed in the most recent commit, can you take another look @mio-stripe ?